# HG changeset patch
# User Olaf Hering <olaf@aepfle.de>
# Date 1394470217 0
# Node ID 47d070c3483472ba14d21f187c270c32eec5588d
# Parent  45d163e2f5c38f4d252cfd4c05f006a9e69217e4
tools/libxc: remove double inclusion of errno.h

xc_private.h includes errno.h already

Signed-off-by: Olaf Hering <olaf@aepfle.de>
Acked-by: Ian Jackson <ian.jackson@eu.citrix.com>

diff -r 45d163e2f5c3 -r 47d070c34834 tools/libxc/xc_compression.c
--- a/tools/libxc/xc_compression.c
+++ b/tools/libxc/xc_compression.c
@@ -32,7 +32,6 @@
 #include <unistd.h>
 #include <sys/types.h>
 #include <inttypes.h>
-#include <errno.h>
 #include "xc_private.h"
 #include "xenctrl.h"
 #include "xg_save_restore.h"
diff -r 45d163e2f5c3 -r 47d070c34834 tools/libxc/xc_flask.c
--- a/tools/libxc/xc_flask.c
+++ b/tools/libxc/xc_flask.c
@@ -19,7 +19,6 @@
 #include "xc_private.h"
 #include <unistd.h>
 #include <stdio.h>
-#include <errno.h>
 #include <fcntl.h>
 #include <string.h>
 #include <sys/mman.h>
diff -r 45d163e2f5c3 -r 47d070c34834 tools/libxc/xc_gnttab.c
--- a/tools/libxc/xc_gnttab.c
+++ b/tools/libxc/xc_gnttab.c
@@ -18,7 +18,6 @@
  */
 
 #include "xc_private.h"
-#include <errno.h>
 
 int xc_gnttab_op(xc_interface *xch, int cmd, void * op, int op_size, int count)
 {
diff -r 45d163e2f5c3 -r 47d070c34834 tools/libxc/xc_minios.c
--- a/tools/libxc/xc_minios.c
+++ b/tools/libxc/xc_minios.c
@@ -29,7 +29,6 @@
 #include <mini-os/events.h>
 #include <mini-os/wait.h>
 #include <sys/mman.h>
-#include <errno.h>
 
 #include <xen/memory.h>
 #include <unistd.h>
diff -r 45d163e2f5c3 -r 47d070c34834 tools/libxc/xc_pm.c
--- a/tools/libxc/xc_pm.c
+++ b/tools/libxc/xc_pm.c
@@ -19,7 +19,6 @@
  *
  */
 
-#include <errno.h>
 #include <stdbool.h>
 #include "xc_private.h"
 
diff -r 45d163e2f5c3 -r 47d070c34834 tools/misc/xen-hptool.c
--- a/tools/misc/xen-hptool.c
+++ b/tools/misc/xen-hptool.c
@@ -1,7 +1,6 @@
 #include <xenctrl.h>
 #include <xc_private.h>
 #include <xc_core.h>
-#include <errno.h>
 #include <unistd.h>
 
 #define ARRAY_SIZE(a) (sizeof (a) / sizeof ((a)[0]))
diff -r 45d163e2f5c3 -r 47d070c34834 tools/misc/xen-mfndump.c
--- a/tools/misc/xen-mfndump.c
+++ b/tools/misc/xen-mfndump.c
@@ -1,7 +1,6 @@
 #include <xenctrl.h>
 #include <xc_private.h>
 #include <xc_core.h>
-#include <errno.h>
 #include <unistd.h>
 #include <inttypes.h>
 
